src/force-app/main/default/permissionsets/Access_Customer_Support_App.permissionset-meta.xml

```diff
@@ -4,6 +4,14 @@
         <application>Customer_Support</application>
         <visible>true</visible>
     </applicationVisibilities>
+    <classAccesses>
+        <apexClass>ContactSelector</apexClass>
+        <enabled>true</enabled>
+    </classAccesses>
+    <classAccesses>
+        <apexClass>CreateCaseActionController</apexClass>
+        <enabled>true</enabled>
+    </classAccesses>
```

Comment:
I would suggest updating Profile instead. Also, this update does not reflect all Apex changes.

15 mins added.